src/XMakeBuildEngine/BackEnd/BuildManager/BuildParameters.cs

```diff
@@ -584,6 +590,15 @@ public bool OnlyLogCriticalEvents
         }
 
         /// <summary>
+        /// A list of warnings to treat as errors.  To treat all warnings as errors, set this to an empty <see cref="HashSet{String}"/>.  
```

Comment:
Hmm. The double-use here seems a bit mysterious, in a way it doesn't on the command line. But it's not *wrong* . . .